api/models.py

```diff
@@ -14,11 +14,23 @@ def __str__(self):
         return self.category_name
 
 
+class AssetSubCategory(models.Model):
+    """Stores all asset sub categories"""
+    sub_category_name = models.CharField(max_length=40, null=False)
+    created_at = models.DateTimeField(auto_now_add=True, editable=False)
+    last_modified = models.DateTimeField(auto_now_add=True, editable=False)
+
+    def __str__(self):
+        return self.sub_category_name
+
+
 class AssetType(models.Model):
     """Stores all asset types"""
     asset_type = models.CharField(max_length=500)
     created_at = models.DateTimeField(auto_now_add=True, editable=False)
     last_modified = models.DateTimeField(auto_now=True, editable=False)
+    asset_sub_category = models.ForeignKey(AssetSubCategory,
+                                           on_delete=models.CASCADE, default=1)
```

Comment:
not sure we should be assigning a default sub-category

Reply:
I did not want to assign a default but I got this prompt when I did not assign a default subcategory
<img width="1214" alt="screen shot 2018-03-01 at 12 06 54" src="https://user-images.githubusercontent.com/6042152/36843087-d26c1456-1d5e-11e8-9677-8d421b1a2967.png">
I got this prompt because I moved `AssetSubCategory` before `AssetType`. If I did not do this then I could not add the foreign key

Reply:
that happens when making a change to model relationships, but the one-off default should be provided on the console for the migration. if done on the model it can cause undesired effects.

Reply:
Alright makes sense, I will change this.